assets/js/script.js

```diff
@@ -43,7 +45,10 @@ input.addEventListener('keypress', function (event) {
     if (event.keyCode === 13) { // 13 is the keyCode for the enter key
         addButton.click(); // this will add a word upon clicking enter
     }
-})
+});
+
+// Call on renderGroceryItems when the page loads
+renderGroceryItems();
```

Comment:
This seems like the same functionality as line 38, but as long as it's still working as expected we might be good.